### ISSUE TITLE
Baseline task data products to be written in CSV format

### ISSUE THREAD

**Issue by chinmaib** [ml4ai/tomcat#312]

**Comment by CalebUAz** [comment]
I made changes to the affective task when it comes to logging data. 

![image](https://user-images.githubusercontent.com/90938130/167000446-d011c841-c0ae-4eaa-8c58-b2f7c9fbfa66.png)

A. So the first row in both team and individual task it logs as soon the first image pops on the subject screen. 
B. Similarly for team and individual task, it logs every action taken by the subjects. 
C. Similarly for team and individual task, when the subject hits submit it logs the final result. 

@adarshp @kobus-barnard is this what you envisioned during the Mission's meeting?

**Comment by adarshp** [comment]
Hi @CalebUAz 

- Instead of having a blank value for the `submit` column, can you change the column name to `event_type`, and choose three sensible values for the cases A, B, and C that you described? (e.g., `first_image`, `intermediate_selection`, `final_submission` - or better names if you can think of them)
- For the human readable timestamp, I was hoping we could use ISO-8601 UTC timestamps to be consistent with the rest of the testbed timestamps. You can do this in Python - see below for an example.

```python
from datetime import datetime
timestamp = datetime.utcnow().isoformat() + "Z"
```

**Comment by CalebUAz** [comment]
Sure I can do that @adarshp

**Comment by eduongAZ** [comment]
I don't think "boot time" is helpful. We should probably remove it.

**Comment by kobus-barnard** [comment]
I have discussed end point data and formats with Emily, following Friday's discussion with Caleb and Eric. Here are some comments and requests:

1) We will err on the side of redundancy and self-documenting. To review, the goal of the CSV files is an intermediate representation that supports easy conversion. 

2) Time: On Friday we discussed a third time format, which was a simplification of the UTC one, but, as the software team is the only group that is likely to see real time, we can do without the third one. Lets go with the two (and improve the headings). In case it helps. I can add that time will generally be massaged by the converters, and end up being something like every 50 milliseconds, with at least some of the data being mapped into that by convolving and sub-sampling. 

3) Picture IDs. We need to record the IAPS number for the picture. If you do not know this, Payal can help. We also need to record our picture number (which is part of the file name, right?). Finally, we need a third column, which is simply the count of the picture seen by that person. This might be the same as the second number, depending on the experiment and is a bit redundant. With all that, it seems do not need the file name. 

4) For the event type, we should have both an integer and the label e.g., "first_image, intermediate_selection, final_submission".  Social scientists will generally use integer categories as their software is much more tuned to that, but not having to look up the meaning of "2" when checking things is probably worth the extra bit of disk space.
